src/ape/cli/choices.py

```diff
@@ -143,15 +171,27 @@ def get_metavar(self, param):
 
 
 class OutputFormat(Enum):
+    """
+    Various output format types to use for outputting data
+    to the user.
```

Comment:
Not particularly descriptive

Reply:
What do you think of 

```
  An enum representing output formats, such as ``TREE`` or ``YAML``.
  Use this to select a subset of common output formats to use
  when creating a :meth:`~ape.cli.choices.output_format_choice`.
```

Reply:
Better. Is there a way to decorate specific options in an enum to describe what they mean?

Reply:
Ya we can document individual enums values, I just added a commit that does that.